test/jdk/java/lang/management/MemoryMXBean/MemoryManagement.java

```diff
@@ -32,7 +32,9 @@
  *
  * @modules jdk.management
  * @build MemoryManagement MemoryUtil
- * @run main/othervm/timeout=600 -Xmn8m -XX:+IgnoreUnrecognizedVMOptions -XX:G1HeapRegionSize=1 -XX:-UseLargePages MemoryManagement
+ * @run main/othervm/timeout=600 -Xmn8m -XX:+IgnoreUnrecognizedVMOptions
```

Comment:
Do we want to run this configuration without G1?  Might be separate runs and add requires?

Reply:
It doesn't fail with other GC modes, but I agree that it is misleading. We can make it a separate test and add @requires.